packages/astro/components/Prism.astro

```diff
@@ -39,7 +40,6 @@ if (grammar) {
   html = Prism.highlight(code, grammar, lang);
 }
 
-let className = lang ? `language-${lang}` : '';
 ---
 
-<pre class={className}><code class={className}>{html}</code></pre>
+<pre class={className}><code class={classLanguage} lang={classLanguage}>{html}</code></pre>
```

Comment:
Should the `lang` attribute here be `language-javascript` or just `javascript`? I'm having trouble tracking down Prism docs on this.

Reply:
Nice catch! I think i've misread, apparently, the lang attribute should not be used, since it's a HTML standard for indicating i18n, as in `<html lang="en">`
According to [docs](https://prismjs.com/#basic-usage)  It should be only `class='language-xxxx'`at `code` blocks.

I will fix it and update the PR.